### ISSUE TITLE
Typo in cfg?

### ISSUE THREAD

**Issue by nxdefiant** [ros-drivers/openni2_camera#16]
Hi,

In OpenNI2.cfg, shouldn't it be

``` python
gen.add("ir_mode", int_t, 0, "Video mode for IR camera", 5, 1, 12, edit_method = output_mode_enum)
```

instead of

``` python
gen.add("ir_mode", int_t, 0, "Video mode for IR camera", 1, 5, 12, edit_method = output_mode_enum)
```

because the order of the values is default, min, max?

**Comment by jkammerl** [comment]
Thanks for pointing this out! I fixed it in: https://github.com/ros-drivers/openni2_camera/commit/5cefdd106b9ddb1a49d5bb5710c5d6f204798fd7

**Comment by mikeferguson** [comment]
I'll get a new release out this afternoon.

**Comment by mikeferguson** [comment]
Released in 0.1.1
